synapse/config/saml2_config.py

```diff
@@ -225,11 +221,14 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
         # At least one of `sp_config` or `config_path` must be set in this section to
         # enable SAML login.
         #
-        # (You will probably also want to set the following options to `false` to
+        # You will probably also want to set the following options to `false` to
         # disable the regular login/registration flows:
         #   * enable_registration
         #   * password_config.enabled
         #
+        # You will also want to investigate the settings under the sso configuration
```

Comment:
```suggestion
        # You will also want to investigate the settings under the "sso" configuration
```

With "configuration section below", I was initially a little confused about whether you were talking about `saml2_config:` :upside_down_face: